deployment/debian-x64/package.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+source ../build.common.sh
+
+VERSION=`get_version ../..`
+
+package_portable ../.. `pwd`/dist/jellyfin_${VERSION}
+
+# Build a Jellyfin .deb file with Docker on Linux
+# Places the output .deb file in the parent directory
+
+package_temporary_dir="`pwd`/pkg-dist-tmp"
+output_dir="`pwd`/pkg-dist"
+current_user="`whoami`"
+image_name="jellyfin-debuild"
+
+cleanup() {
+    set +o errexit
+    docker image rm $image_name --force
+    rm -rf "$package_temporary_dir"
+}
+trap cleanup EXIT INT
+
+docker build ../.. -t "$image_name" -f ./Dockerfile --build-arg SOURCEDIR="/jellyfin-${VERSION}"
+mkdir -p "$package_temporary_dir"
+mkdir -p "$output_dir"
+docker run --rm -v "$package_temporary_dir:/temp" "$image_name" sh -c 'find / -maxdepth 1 -type f -name "jellyfin*" -exec mv {} /temp \;'
+chown -R "$current_user" "$package_temporary_dir"
```

Comment:
Actually, I'd also like it to not do such a broad find on `/` in the container.

I think this is a consequence of trying to use this Dockerfile to do the generic binary as well. As mentioned in my previous comment, we should split that out, so there wouldn't be any need to alter the flow from how the original `build-deb.sh` script does it, i.e. copying just the files from `/dist`. For now this works though, and I'll do modifications to this later too.

Reply:
Well it only copied from /dist because it hard codes a move of just the binary package. Since we also want the source package and desc files we need the find. It's not all that broad since the depth is limited to 1.

I guess I can move the find to the Dockerfile. But it does not make a lot of difference. If you do not want in in the `/` directory the Dockerfile should add an extra layer of directory to `/repo` since the deb will ALWAYS land in `..` this is non configurable (due to all the weird cross dependencies in the tooling.

But in the end it's semantics. I can change it either way.

Reply:
I'm not sure I understand why the `find` is needed, is it not putting all the `dpkg-buildpackage` output into a single place that can be copied as a whole?